sidebarsAstro.js

```diff
@@ -283,24 +286,32 @@ module.exports = {
     },
     {
       type: "doc",
-      label: "Configure the CLI",
-      id: "cli/configure-cli",
+      label: "Develop your project",
+      id: "cli/develop-project",
     },
     {
       type: "doc",
-      label: "Test your Astro project locally",
-      id: "cli/test-your-astro-project-locally",
+      label: "Run your project locally",
+      id: "cli/run-airflow-locally",
     },
     {
       type: "doc",
-      label: "Authenticate to cloud services",
-      id: "cli/authenticate-to-clouds",
+      label: "Test your project",
+      id: "cli/test-your-astro-project-locally",
     },
     {
       type: "doc",
       label: "Release notes",
       id: "cli/release-notes",
     },
+    {
+      type: "category",
+      label: "Advanced configuration",
```

Comment:
```suggestion
      label: "Advanced",
```

Let's just do `advanced` here, no need for the configuration part I don't think